niceman/retrace/tests/test_packagemanagers.py

```diff
@@ -42,6 +43,17 @@ def test_identify_packages():
 
 
 @skip_if(not apt)
+def test_utf8_file():
+    files = [u"/usr/share/ca-certificates/mozilla/TÜBİTAK_UEKAE_Kök_Sertifika_Hizmet_Sağlayıcısı_-_Sürüm_3.crt"]
+    # Simple sanity check that the pipeline works with utf-8
+    packages, origins, files = identify_packages(files)
+    pprint(files)
+    pprint(origins)
+    pprint(packages)
+    assert True
```

Comment:
If the file exists, then we should assume that it came from *some* package. So I can definitely add a test for a reasonable result.